src/browser.js

```diff
@@ -68,7 +70,7 @@ class Gofor {
         this.fetch = (...args) => {
             args[1] = this.mergeOptions(args[1]);
 
-            return this.fetcher.apply(window, args);
+            return this.fetcher.apply(globalScope, args);
```

Comment:
this file is named "browser.js", in which case globalScope is not window?

Reply:
https://github.com/fiverr/gofor/blob/master/src/server.js#L8
![](https://user-images.githubusercontent.com/516342/58259179-b6ee5480-7d7c-11e9-8f42-16ec0af21efd.png)

Reply:
(that was the issue)